src/commands/prompt/index.ts

```diff
@@ -112,10 +129,20 @@ async function runInternal(initialPrompt: string, options: PromptOptions) {
     throw new Error(`Provider config not found: ${provider.name}.`);
   }
 
+  let style = {};
+
+  if (options.creative) {
+    style = RESPONSE_STYLE_CREATIVE;
+  }
+  if (options.precise) {
+    style = RESPONSE_STYLE_PRECISE;
+  }
+
```

Comment:
Handle case when user sets both, let's warn them in such case.

Reply:
Done